### PR TITLE
Use goreleaser to release.

### LINKED ISSUE #197: Create new releases entirely in Travis CI
Currently when releasing Centrifugo I create binary releases manually at my computer and then upload them on Github release page.

To create binary releases at moment:

```
make release
```

– is used.

It would be great if in Travis CI we not only upload deb/rpm packages to packagecloud.io but also create binary Github releases. https://docs.travis-ci.com/user/deployment/releases/

### PR DESCRIPTION
**Pull request by furdarius**
Simplify release workflow using [goreleaser](https://github.com/goreleaser/goreleaser).
Build with `-s -w` flags to minify output file.
Use alpine:3.7 as FROM Docker image.

Fixes #197

Signed-off-by: Artemiy Ryabinkov <getlag@ya.ru>

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  - CHANGELOG*
+  format_overrides:
+    - goos: windows
+      format: zip
+nfpm:
+  name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm
+    }}v{{ .Arm }}{{ end }}'
+  homepage: https://github.com/centrifugal/centrifugo
+  maintainer: Alexander Emelin <frvzmb@gmail.com>
+  description: Centrifugo - real-time websocket messaging server.
```
Comment (FZambia) [inline .goreleaser.yml]: `Centrifugo – real-time messaging server.`

File: .goreleaser.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  license: MIT
+  formats:
```
Comment (FZambia) [inline .goreleaser.yml]: let's only create binary releases at moment without `deb` and `rpm` packages. We already have separate step for `deb` and `rpm` uploaded on packagecloud in Travis CI.

File: .goreleaser.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  commit_author:
+    name: Alexander Emelin
+    email: frvzmb@gmail.com
+  install: bin.install "centrifugo"
+  skip_upload: true
+builds:
+- goos:
+  - linux
+  - darwin
+  - windows
```
Comment (FZambia) [inline .goreleaser.yml]: Is it possible to build for exactly same platforms we already have in releases without windows 386? I.e:
* darwin amd64
* freebsd amd64
* linux 386
* linux amd64
* linux arm
* windows amd64

File: .goreleaser.yml (hunk 2 above)
Comment (FZambia) [reply .goreleaser.yml]: We still can add them later if needed

**Comment by FZambia** [comment]
Thank you very much @furdarius ! Added some comments on changes.

File: .goreleaser.yml (hunk 1 above)
Comment (furdarius) [reply .goreleaser.yml]: Fixed

File: .goreleaser.yml (hunk 2 above)
Comment (furdarius) [reply .goreleaser.yml]: Commented

File: .goreleaser.yml (hunk 3 above)
Comment (furdarius) [reply .goreleaser.yml]: No problem! Done!

**Comment by furdarius** [comment]
@FZambia glad to help you!
All review notes fixed.

**Comment by FZambia** [comment]
Thanks!

**Pull request merged**